### ISSUE TITLE
AutoPkgr fails to update Munkitools to 5.2.0.4240

### ISSUE THREAD

**Issue by jby** [lindegroup/autopkgr#659]
AutoPkgr says at launch that Munkitools 5.2.0.4240 is available even though I have it installed on the machine where AutoPkgr is running.
When giving AutoPkgr the oportunity to upgrade it runs for a while, then ends in an error saying "Repo is not installed"

**Comment by dev-yeet** [comment]
I'm getting this as well. Even if you manually update to the latest version, it still prompts that a new update is available.

**Comment by mikafletcher** [comment]
Hello, I have the same problem, have you found a solution ?

**Comment by Prodeguerriero** [comment]
Similar problem here. I even tried to remove `munkitools` and installed it again. I still heard the error sound when installing, and AutoPkgr is still convinced I need to upgrade to the latest version.

**Comment by jlgore** [comment]
I also ran into this issue, and while troubleshooting I installed the most recent `munkitools`. Even after that Autopkgr didn't want to detect the latest version was installed. As I was trying to sift through logs today to troubleshoot further, my 900th attempt at clicking install worked and it sees the latest tools. I don't know if I have any logs that would be useful but I am down to post them if it's helpful.
